lib/collection/src/operations/config_diff.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(deprecated)] // hack to remove warning for memmap_threshold deprecation below
+
```

Comment:
A bit of a hack, sadly.

This is required to satisfy the compiler for the `memmap_threshold` field deprecation. If we remove this, it'll complain about the field itself.

The alternative would be to not mark the field as deprecated. But that would also remove the deprecation flag in OpenAPI.